hacking/test-module

```diff
@@ -152,15 +153,15 @@ def boilerplate_module(modfile, args, interpreters, check, destfile):
         task_vars=task_vars
     )
 
-    if module_style == 'new' and 'ANSIBALLZ_WRAPPER = True' in module_data:
+    if module_style == 'new' and 'ANSIBALLZ_WRAPPER = True' in to_text(module_data):
```

Comment:
Sorry, I was unclear.  For this one I meant, just use bytes all the way through.  Don't convert to text anywhere:

``` python
if module_style == 'new' and b'ANSIBALLZ_WRAPPER = True' in module_data:
```